### ISSUE TITLE
JXTaskPane Support

### ISSUE THREAD

**Issue by Reap00r** [atarw/material-ui-swing#57]
The Material LaF is very nice, thanks for doing this. I'm currently working on a product which uses a SwingX Component JXTaskPane. Would it be possible to add support for this?

**Comment by vincenzopalazzo** [comment]
@Reap00r Hello, Does not know this library but taking a look at the code of the project open surse I know that you can not support it in our look and feel because you should also change the swingx library.

But we wait @atarw and see if he agrees with me or I said a bullshit


in the meantime, this is what made me say this -> [Click me](https://github.com/ebourg/swingx/tree/master/swingx-plaf/src/test/java/org/jdesktop/swingx/plaf)

**Comment by Reap00r** [comment]
Hi @vincenzopalazzo, thanks for the quick resonse.

Sure, we can wait on @atarw for his reply.
I took a look at the UI myself and thought it might be possible to override the defaults without any changes to the swingx library, but maybe I'm wrong and I missed something ;)

The way you currently set the UI is something like this:
table.put("TableHeaderUI", tableHeaderUI);
So wouldn't it be possible to write a JXTaskPaneUI which can be added to the (UIDefaults) table?
The TaskPane then loads your UI not the default one?!

Additionally, I saw that we also need support for the JXTaskPaneContainer. Should I open another issue or should I just mention it in here?

Thanks for your help!

**Comment by vincenzopalazzo** [comment]
Hi @Reap00r,

I try to look the code and i try a implement a solution

**Comment by vincenzopalazzo** [comment]
@Reap00r I have a news, 

This are a results for my first test, and I have a question for you, **You know which component draws the collapsible panel (the celestial one), I tried to look at the swingx code but I just can not find it?**

This a init a work
![selection_010](https://user-images.githubusercontent.com/17150045/53180141-53298380-35f5-11e9-8b2e-e8fddd0816bb.png)
![selection_011](https://user-images.githubusercontent.com/17150045/53180153-57ee3780-35f5-11e9-8eab-910ba553016f.png)

**Comment by Reap00r** [comment]
@vincenzopalazzo Thanks for your update.
Unfortunately I'm currently not at work and can't take a look at our code. I will try to help you investigate tomorrow and will let you know if I find something.

**Comment by vincenzopalazzo** [comment]
@Reap00r This is end work, sorry for my instability but I'm tired but I only finished the exam session last night, and I'm sorry if I was surly and I said that you could not support the component because it was not part of swing, sorry.

![finalclose](https://user-images.githubusercontent.com/17150045/53187949-89223400-3604-11e9-9e10-1ea751d1a310.png)
![finalopen](https://user-images.githubusercontent.com/17150045/53187955-8cb5bb00-3604-11e9-8a40-cbaa2724f9e2.png)

 **In the next hours I create a pull request**

**Comment by atarw** [comment]
Sorry haven't been active on Github recently - thank you @vincenzopalazzo for fixing this issue!
